autogpt_platform/backend/backend/data/execution.py

```diff
@@ -492,20 +492,25 @@ async def update_graph_execution_stats(
     data = stats.model_dump() if stats else {}
     if isinstance(data.get("error"), Exception):
         data["error"] = str(data["error"])
-    res = await AgentGraphExecution.prisma().update(
-        where={
-            "id": graph_exec_id,
-            "OR": [
```

Comment:
Could we do execution status in [enums]?

Reply:
If the additional clause is indeed ignored, let's just use update_many return None if the result is < 1 else return the fund_unique result.

Reply:
fixed